engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -318,7 +345,7 @@ public void preRenderSetupLightGeometry() {
         glDisable(GL_DEPTH_TEST);
 
         glEnable(GL_BLEND);
-        glBlendFunc(GL_ONE, GL_ONE);
+        glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_COLOR);
```

Comment:
This is what helps multiple dropped lights not to cause overbrightness.

Before (20 torches):
![terasology-160408094050-1280x720](https://cloud.githubusercontent.com/assets/5922109/14394176/c7a90886-fd7f-11e5-9f8a-c9b879dde548.jpg)

After:
![terasology-160408114948-1280x720](https://cloud.githubusercontent.com/assets/5922109/14394255/340f0188-fd80-11e5-8831-4b9dd94b5fd6.jpg)